assets/js/modules/tagmanager/setup.js

```diff
@@ -228,17 +242,26 @@ class TagmanagerSetup extends Component {
 
 			this.validateAccounts( accounts, selectedAccount );
 
-			// If the selectedContainer is not in the list of containers, clear it.
-			if ( isValidContainerID( selectedContainer ) && ! containers.find( ( container ) => container.publicId === selectedContainer ) ) { /* Capitalization rule exception: `publicId` is a property of an API returned value. */
-				selectedContainer = null;
+			// If the selected container is not in the list of containers, clear it.
+			const containerIDs = containers.map( ( { publicId } ) => publicId ); /* Capitalization rule exception: `publicId` is a property of an API returned value. */
+			if ( isValidContainerID( selectedContainer ) && ! containerIDs.includes( selectedContainer ) ) {
+				selectedContainer = '';
 			}
+			if ( isValidContainerID( selectedContainerAMP ) && ! containerIDs.includes( selectedContainerAMP ) ) {
+				selectedContainerAMP = '';
+			}
+
+			const containersWeb = getContainers( containers ).byContext( USAGE_CONTEXT_WEB );
```

Comment:
This is named differently to not conflict with `containers` which is already defined in the current scope.